sites/friday/src/manage-resources/adjust-resources.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Resource configuration
+description: Configure CPU, RAM, and disk storage on a per-environment basis so your apps and services can run smoothly.
+weight: -100
+keywords:
+  - "resources"
+  - "flexible resources"
+  - "CPU"
+  - "RAM"
+  - "disk storage"
+  - "horizontal scaling"
+  - "vertical scaling"
+  - "scaling"
+---
+
+When you first deploy your project or add a new app or service to it,
+{{% vendor/name %}} allocates [default resources](/manage-resources/resource-init.md#default-resources) to each of your containers.
+If you don't want to use those default resources, define a different [resource initialization strategy](/manage-resources/resource-init.md#define-a-resource-initialization-strategy).
+
+After the initial deployment, or if you opt for the `Manual` [resource initialization strategy](/manage-resources/resource-init.md#define-a-resource-initialization-strategy),
+you can adjust container resources manually.
+To do, follow the instructions on this page.
+
+{{% vendor/name %}} allows you to configure resources (CPU, RAM, and disk) per environment for each of your apps and services.
+You can also add instances for each of your apps depending on your needs.
+
+For example, you can scale vertically and allocate more resources to your production and staging environments
+than to your development environments.
+This flexibility allows you to optimize performance and costs.
+
+You can even scale horizontally if your apps are struggling with high load, or if you're expecting a traffic spike,
+by adding more instances for your apps and workers.
+
+For information on costs related to resource usage, see the [{{% vendor/name %}} pricing page](https://upsun.com/pricing/).
+To keep an eye on those costs in the Console, see [Resource Billing](/manage-resources/resource-billing.md).
+
+## Vertical scaling
+
+To define how much CPU, RAM, and disk storage you want to allocate to each individual container,
+follow these steps:
+
+{{< codetabs >}}
+
++++
+title= Using the CLI
++++
+
+Run the `{{% vendor/cli %}} resources:set` command, and follow the prompts to set resources for each of your apps and services.
+
+{{< note title= "Tip" >}}
+
+For further guidance on how to set resources using the CLI, run the `{{% vendor/cli %}} resources:set --help` command.
+
+{{< /note >}}
+
+After you've set resources, your environment is redeployed,
+which causes a short downtime.
+
+<--->
+
++++
+title= From the Console
++++
+
+1. Navigate to your environment.
+2. Select an app or service in the tree on the left-hand side:
```

Comment:
We added some new buttons to open the `Configure resources` page, that are a bit more easily accessible.

You can now open the `Configure resources` page from the environment card (or project card) directly:
![Screenshot 2023-12-05 at 5 15 41 PM](https://github.com/platformsh/platformsh-docs/assets/12087186/a0464075-6946-41a8-9e4b-65a4a68bfaeb)

Or from the `Apps & Services` graph too, but without having to open a specific app or service:
![Screenshot 2023-12-05 at 5 15 52 PM](https://github.com/platformsh/platformsh-docs/assets/12087186/7b9a8578-13d4-40cc-b6b7-9459176ade12)

I think both of these buttons offer an easier path than the one we describe currently.
Could we open the doc accordingly?
Let me know if you need a cleaner screenshot.